src/utils/calendar.ts

```diff
@@ -22,7 +22,7 @@ export const createCalendar = ({
       organizer: `${event.host || "Unknown"} <undefined>`, // string format: "name <email>". email cannot be blank
       description: event.description,
       summary: event.name,
-      url: getFullVenueInsideUrl(event.venueId),
+      url: getAbsoluteAttendeeSpaceInsideUrl(event.worldSlug, event.venueSlug),
```

Comment:
Maybe for better DX:

```suggestion
      url: generateSpaceInsideUrl( { worldSlug: event.worldSlug, spaceSlug: event.venueSlug, absolute: true }),
```

Reply:
I like.

Reply:
Resolved in a1a8ad822957ff7b46c3b7b881e87b113f27145c